package.json

```diff
@@ -93,4 +93,4 @@
     ],
     "testURL": "http://localhost/"
   }
-}
+}
```

Comment:
Would you please add again the newline? :)

Reply:
Sorry, returned